README.md

```diff
@@ -89,7 +89,9 @@ This doc assumes that
 	* [acmeair-authservice-java](https://github.com/blueperf/acmeair-authservice-java)
 	* [acmeair-customerservice-java](https://github.com/blueperf/acmeair-customerservice-java)
 	* [acmeair-flightservice-java](https://github.com/blueperf/acmeair-flightservice-java)
-	
+* **Note:** After importing every project, checkout the branch to **microclimate-mp-1.3** from theia editor terminal using the command below
+	* git checkout microclimate-mp-1.3
```

Comment:
Suggested change:
**Note:** After you import each project, use the `git checkout microclimate-mp-1.3` command from the Theia editor terminal to check out the branch to `microclimate-mp-1.3`.


Reasons for the change:
- For accessibility, we need to remove words like "below" because a visually impaired user might not be able to understand them.
- "Check out" is two words as a verb.